project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.3.9
+sbt.version=1.3.6
```

Comment:
does not build on intellij without moving the version back down

Reply:
It builds fine for me in IntelliJ.  I'm running an older version, 2019.3.1 Ultimate.  I can try the most up-to-date version on my other machine.

What error do you get?